projects/router/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ngx-matomo/router",
-  "version": "3.2.0",
+  "version": "4.0.0",
```

Comment:
Version will be bumped automatically when merged, so it should not be manually changed.